htdocs/profile.bml

```diff
@@ -742,6 +743,18 @@ body<=
         @posting_access_to_userids, @posting_access_from_userids,
     );
 
+    # pull in list of banned users
+    my $banned = LJ::load_rel_user($u, 'B') || [];
+    my %banned_users = map { $_ => 1 } @$banned;
+
+    # remove banned users from lists to display
+    @$_ = grep ( !defined $banned_users{ $_ }, @$_ )
```

Comment:
You probably want `exists` here instead of `defined`.

Also, our style is to use `grep { }` (with braces instead of parenthesis).